test/fennec/udp_test.exs

```diff
@@ -228,8 +230,12 @@ defmodule Fennec.UDPTest do
   end
 
   defp udp_communicate(udp, client_id, req) do
+    with_mock Fennec.Auth, [:passthrough], [
```

Comment:
Because tests would be more complicated. It's impossible to setup tests to connect with server at first try, due to random `nonce`. `nonce` "negotiation" makes testing more complicated then it should be.